src/Codeception/Configuration.php

```diff
@@ -500,6 +500,58 @@ public static function projectDir()
     }
 
     /**
+     * Returns path to a given directory relative to your project.
+     * @param string $path
+     * @return string
+     */
+    public static function getRelativeDir($path)
+    {
+        $relPath = $path;
+        if (substr($path, 0, strlen(self::projectDir())) == self::projectDir()) {
+            $relPath = substr($path, strlen(self::projectDir()));
+        } else {
+            $pathPrefix = self::projectDir();
+            $sameAbsoluteness = true;
+            $rootPathIndex = 0;
+            if (DIRECTORY_SEPARATOR == '\\') {
+                $hasDriveSpecifier = preg_match('/^[A-Za-z]:/', $pathPrefix);
```

Comment:
I know there are a couple regexes... This one matches Windows style drive letters.